beacon-chain/rpc/eth/helpers/sync.go

```diff
@@ -20,11 +20,31 @@ func ValidateSync(ctx context.Context, syncChecker sync.Checker, headFetcher blo
 		return nil
 	}
 	headSlot := headFetcher.HeadSlot()
+	// optimisticModeFetcher := TODO
+	// beaconServer := TODO -> ctx.beaconServer?
+	// beaconState, err := beaconServer.StateFetcher.State(ctx)
+	// if err != nil {
+	// 	return status.Errorf(
+	// 		codes.Internal,
+	// 		"Could not fetch state from beacon server: %v",
+	// 		err,
+	// 	)
+	// }
+	isOptimistic, isOptimisticErr := IsOptimistic(ctx, nil, nil)
```

Comment:
We typically name error variable the same way `err`

```suggestion
	isOptimistic, err := IsOptimistic(ctx, nil, nil)
```

https://earthly.dev/blog/golang-errors/